### PR TITLE
Web design

### PR DESCRIPTION
**Pull request by Rangatin**

### CODE REVIEW

File: FromLocalsToLocals/Views/Home/Reviews.cshtml [hunk 1]
```diff
@@ -4,55 +4,140 @@
...
+                            1003 reviews
+                        </div>
+                        <ul class="list-group list-group-flush">
+                            <!-- 5 star reviews-->
+                            <li class="list-group-item">
+                                <span class="fa fa-star checked"></span>
+                                <span class="fa fa-star checked"></span>
+                                <span class="fa fa-star checked"></span>
+                                <span class="fa fa-star checked"></span>
+                                <span class="fa fa-star checked"></span>
```
Comment (HabaHaba05) [inline FromLocalsToLocals/Views/Home/Reviews.cshtml:83]: I believe, it's possible to write it using a loop:
`@for(var i =0;i<5;i++){
<span class="fa fa-star checked"></span>
}`

**Review by StauceLukas — CHANGES_REQUESTED** [review]
Merge conflict with _Layout.cshtml. Something went wrong

File: FromLocalsToLocals/Controllers/ReviewsController.cs [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            db.SaveChanges();
+        }
+
+        private int GetIndex(FrameworkElement element)
+        {
+            return RView.Items.IndexOf(element.DataContext);
+        }
+    } 
+         */
+    }
```
Comment (HabaHaba05) [inline FromLocalsToLocals/Controllers/ReviewsController.cs:190]: All this code can be taken to ReviewsViewModel (it need to be created in the model folder I think) I believe, in order to keep the controller tidy.

Comment (Rangatin) [reply FromLocalsToLocals/Controllers/ReviewsController.cs:190]: Indeed. I pasted and commented out this code for further usage. It's incorrectly placed - that's why it's all a one big coment;)

File: FromLocalsToLocals/Views/Home/Reviews.cshtml (hunk 1 above)
Comment (Rangatin) [reply FromLocalsToLocals/Views/Home/Reviews.cshtml:83]: Didn't know about that. Let me check.

**Review by HabaHaba05 — APPROVED** [review]

File: FromLocalsToLocals/Views/Home/Reviews.cshtml (hunk 1 above)
Comment (Rangatin) [reply FromLocalsToLocals/Views/Home/Reviews.cshtml:83]: It can be applied only to 5 stars, if that's what you have in mind;)

**Review by StauceLukas — APPROVED** [review]

**Pull request merged**